packages/cli/src/commands/doctor/runAutomaticFix.ts

```diff
@@ -1,38 +1,41 @@
-// @flow
 import chalk from 'chalk';
-import ora from 'ora';
+import ora, {Ora} from 'ora';
 import {logger} from '@react-native-community/cli-tools';
-// $FlowFixMe - converted to TS
 import {HEALTHCHECK_TYPES} from './healthchecks';
-import type {EnvironmentInfo} from './types';
+import {EnvironmentInfo, HealthCheckCategoryResult} from './types';
 
-const AUTOMATIC_FIX_LEVELS = {
-  ALL_ISSUES: 'ALL_ISSUES',
-  ERRORS: 'ERRORS',
-  WARNINGS: 'WARNINGS',
-};
+enum AUTOMATIC_FIX_LEVELS {
+  ALL_ISSUES = 'ALL_ISSUES',
+  ERRORS = 'ERRORS',
+  WARNINGS = 'WARNINGS',
+}
 
 export {AUTOMATIC_FIX_LEVELS};
-export default async ({
+
+interface RunAutomaticFixArgs {
+  healthchecks: HealthCheckCategoryResult[];
+  automaticFixLevel: AUTOMATIC_FIX_LEVELS;
+  stats: {
+    errors: number;
+    warnings: number;
+  };
+  loader: Ora;
+  environmentInfo: EnvironmentInfo;
+}
+
+export default async function({
   healthchecks,
   automaticFixLevel,
   stats,
-  loader,
```

Comment:
I saw here that this function doesn't use its `loader` argument, but uses `ora` directly from the import. Is this intended?

Reply:
if loader is unused, please remove it from the type def as well